app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,36 @@
 <resources>
     <string name="app_name">FaceAccess Client</string>
+    <string name="title_activity_settings">Settings</string>
+
+    <!-- Preference Titles -->
+    <string name="messages_header">Messages</string>
+    <string name="sync_header">Sync</string>
+
+    <!-- Messages Preferences -->
+    <string name="signature_title">Your signature</string>
+    <string name="reply_title">Default reply action</string>
```

Comment:
I don't understand the need for these lines :confused:

Reply:
removed